homepage/sv.js

```diff
@@ -2,66 +2,75 @@
+	"Chat, for GitHub.": "Chatt, för GitHub.",
+	"Get unlimited public rooms and one-to-one chats, free forever. Private team plans start from $2 per user per month. No credit card required.": "Få oändligt med öppna rum och privata en-och-en rum, gratis för alltid. Privata prisplaner startar på $2 per användare varje månad. Inget kreditkort behövs.",
 	"Sign up free with Github": "Registrera dig gratis med GitHub",
-	"Sign up with Github": "Registrera dig med github",
-	"By signing up you agree to our <a %s>Terms and Conditions</a>": "genom att registrera dig godkänner du våra <a %s>Villkor</a>",
-	"Markdown power": "Markdown power",
-	"Ever wanted full markdown support in chat? How about syntax highlighting?": "någonsin önskar full markdown i chattar? Vad sägs som syntax highlighting?",
-	"Yip, we got your back.": "Jup, Det har vi.",
+	"Sign up with Github": "Registrera dig med Github",
+	"By signing up you agree to our <a %s>Terms and Conditions</a>": "Genom att registrera dig godkänner du våra <a %s>Terms and Conditions</a>",
+	"Markdown power": "Textformatering makt",
+	
+	"Ever wanted full markdown support in chat? How about syntax highlighting?": "Någonsin önskat full textformatering i chattar? Vad sägs som syntax markeringar?", 
+	"Yip, we got your back.": "Yepp, det har vi.", 
 	"Private conversations": "Privata konversationer",
-	"Public rooms are awesome (and free), but we get that some stuff is private. So we've got <a %s>plans for that</a>.": "Publika rum är super (och gratis), men vi fattar att vissa saker är privata. Så vi har <a %s>prisplaner för det/a>.",
+	"Public rooms are awesome (and free), but we get that some stuff is private. So we've got <a %s>plans for that</a>.": "Offentliga rum är häftiga (och gratis), men vi fattar att vissa saker är privata. Så vi har <a %s>prisplaner för det/a>.",
 	"Developer friendly": "Utvecklarvänligt",
-	"This is a picture of a cloud against a background of clouds. OMG! We <a %s>integrate</a> with a bunch of stuff in the cloud.": "detta är en bild av ett moln, mot en bakgrund av moln. OMG! Vi <a %s>Integrerar</a> med en del grejer i molnet.",
+	"This is a picture of a cloud against a background of clouds. OMG! We <a %s>integrate</a> with a bunch of stuff in the cloud.": "Detta är en bild på ett moln, mot en bakgrund av moln. OMG! Vi <a %s>Integrerar</a> med massor i molnet.",
 	"Some of our communities": "Några av våra grupper",
-	"Create your own": "Skapa din egen",
+	"Create your own": "Skapa din egna",
 	"Explore rooms": "Utforska rum",
 	"Explore": "Utforska",
 	"Integrated with your workflow": "Integrerad med ditt arbetsflöde",
-	"Gitter is built on top of GitHub and is tightly integrated with your organisations, repositories, issues and activity.": "Gitter är byggt på GitHub och är nära integrerat med dina organisationer, repositories, issues och aktivitet.",
```

Comment:
'Repositories' & 'Issues' is being skipped as it doesn't have a 'good official translation', most people know what it means anyways as it's GitHub related